generate-component/index.js

```diff
@@ -0,0 +1,77 @@
+const fs = require("fs");
```

Comment:
@somethiiing can we use `import` here and `.ts`? You can use tsx to run the file:

```
npx tsx generate-component
```